examples/knative/powershell/kn-ps-vrni-databus/function.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-ps-vrni-databus
+  labels:
+    app: veba-ui
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      imagePullSecrets:
+        - name: regcred
+      containers:
+        - image: vmwsmitmartijn/kn-ps-vrni-databus:1.1
```

Comment:
The `image` name be: `us.gcr.io/daisy-284300/veba/kn-ps-vrni-databus1.0` and once this is approved, we'll build/push the image.

Reply:
@smitmartijn please change this line as per @lamw's comment since we can't change it after the fact. I'll build and push the image once the PR is merged.

Reply:
Done! I thought this had to be done after pushing it to the registry.